package.json

```diff
@@ -68,6 +68,7 @@
     "@neutrinojs/jest": "9.0.0",
     "@testing-library/jest-dom": "5.0.2",
     "@testing-library/react": "9.3.3",
+    "markdownlint-cli": "0.22.0",
```

Comment:
What's the reason you're adding the package via `package.json`?

Reply:
You can still run `pre-commit run --all-files markdownlint-cli` without having to add the dependency.

Reply:
I just thought of installing it via package to avoid installation while running pre-commit. I will remove this from the package. The former one is a better way.